crates/telio-traversal/src/endpoint_state.rs

```diff
@@ -0,0 +1,120 @@
+use std::fmt;
+use telio_utils::telio_log_warn;
+
+#[derive(Debug, Default, PartialEq)]
+pub enum EndpointState {
```

Comment:
Since this doesn't hold any data, I think you should derive `Copy` (and `Clone`) and pass it around by value instead of by reference

Reply:
fair enough, will change